README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Overview
 
-Quality is a tool that runs quality checks on code in git repos using
+TODO: Quality is a tool that runs quality checks on code in git repos using
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker